.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+dist: trusty
+language: python
+python:
+  - "3.5"
+
+before_install:
+  - sudo apt-get update -q || true
```

Comment:
See my first link. Sure, the instructions are about getting C++. But strip out the C++ stuff you don't need and use the clang format package from my first link and that should do the trick without relying on Travis's beta container.